AubreyV/asm_system
# added authentication for admin

## Description / Purpose (checkpoints)
This PR implements the Admin Authentication

## Type of change
 - New feature (non-breaking change which adds functionality)

## How To Test This PR
 1. Go to /register
 2. Register an Admin
 3. Log in the newly registered Admin
 4. Check if can logout
 5. Check if **role** in DB is 1

 **Commands**
  - `npm run dev`

 **Routes/URL**
  - `/`
  - `/register`

## Note:
1. Edited file structure
* Added:
** views/admin
** controllers/Admin
** views/auth
** controller/auth

* Moved:
** index.blade.php
** login.blade.php
